Test failed in issue-101082.rs: expected string not found in input

We've started seeing this test failure when trying to package rust for use in chromium ([link to build output](https://logs.chromium.org/logs/chromium/buildbucket/cr-buildbucket/8717326092551615953/+/u/package_rust/stdout?format=raw)). The output says
```
--- stderr -------------------------------
 C:\b\s\w\ir\cache\builder\src\third_party\rust-src\tests\codegen\issues\issue-101082.rs:30:16: error: x86-64-v3: expected string not found in input
  // x86-64-v3: masked.load
...
```

I see the comment in issue-101082.rs mentions that the test got changed until https://github.com/llvm/llvm-project/issues/134513 got fixed upstream, which happened a few hours ago. Perhaps we need to change the test back now?

--- a/tests/codegen/issues/issue-101082.rs
+++ b/tests/codegen/issues/issue-101082.rs
@@ -12,6 +12,7 @@
 // at the time still sometimes fails, so only verify it for the power-of-two size
 // - https://github.com/llvm/llvm-project/issues/134735
 //@[x86-64-v3] only-x86_64
+//@[x86-64-v3] min-llvm-version: 21
 //@[x86-64-v3] compile-flags: -Ctarget-cpu=x86-64-v3
 
 #![crate_type = "lib"]
@@ -19,16 +20,7 @@
 #[no_mangle]
 pub fn test() -> usize {
     // CHECK-LABEL: @test(
-    // host: ret {{i64|i32}} 165
-    // x86-64: ret {{i64|i32}} 165
-
-    // FIXME: Now that this autovectorizes via a masked load, it doesn't actually
-    // const-fold for certain widths.  The `test_eight` case below shows that, yes,
-    // what we're emitting *can* be const-folded, except that the way LLVM does it
-    // for certain widths doesn't today.  We should be able to put this back to
-    // the same check after <https://github.com/llvm/llvm-project/issues/134513>
-    // x86-64-v3: masked.load
-
+    // CHECK: ret {{i64|i32}} 165
     let values = [23, 16, 54, 3, 60, 9];
     let mut acc = 0;
     for item in values {

